server/channels/store/sqlstore/channel_store.go

```diff
@@ -2472,6 +2558,10 @@ func (s SqlChannelStore) UpdateLastViewedAt(channelIds []string, userId string)
 		TotalMsgCountRoot int64
 	}{}
 
+	if len(channelIds) == 0 {
```

Comment:
Not needed anymore (since we are (if I remember correctly) early returning if there are no channelIds) but thought it was a nice to have failsafe.